test/verify/check-storage-resize

```diff
@@ -87,9 +87,9 @@ class TestStorageResize(StorageCase):
             if grow_needs_unmount:
                 self.content_head_action(fsys_row, "Mount")
                 self.confirm()
-                self.wait_mounted(fsys_row, fsys_tab)
             size = int(m.execute("df -k --output=size /run/foo | tail -1").strip())
             assert (size > 300000)
+            self.wait_mounted(fsys_row, fsys_tab)
```

Comment:
This should happen before running `df` so that we can be sure that `df` actually looks at the mounted fsys.

Reply:
Done and reopened